crates/iroha/tests/integration/upgrade.rs

```diff
@@ -333,7 +321,7 @@ fn migration_fail_should_not_cause_any_effects() {
         "failed_migration_test_domain".parse().expect("Valid");
     assert_domain_does_not_exist(&client, &domain_registered_in_migration);
 
-    let _err = upgrade_executor(&client, "../../wasm_samples/executor_with_migration_fail")
+    let _err = upgrade_executor(&client, "executor_with_migration_fail")
         .expect_err("Upgrade should fail due to migration failure");
```

Comment:
```suggestion
    upgrade_executor(&client, "executor_with_migration_fail")
        .expect_err("Upgrade should fail due to migration failure");
```

you just used expect

Reply:
Ah yes, that's why:

<img width="715" alt="image" src="https://github.com/user-attachments/assets/21f16d1d-44df-4cf4-9564-cceb5e2b6647">